networking/routes/route-configuration.adoc

```diff
@@ -49,9 +55,11 @@ include::modules/nw-path-based-routes.adoc[leveloffset=+1]
 
 include::modules/nw-route-specific-annotations.adoc[leveloffset=+1]
 
+ifndef::openshift-rosa,openshift-dedicated[]
 include::modules/nw-route-admission-policy.adoc[leveloffset=+1]
 
 include::modules/nw-ingress-creating-a-route-via-an-ingress.adoc[leveloffset=+1]
+endif::[]
```

Comment:
Just confirming that you want this module `nw-ingress-creating-a-route-via-an-ingress.adoc` out of ROSA/OSD docs. I only ask bc I did not see this section in the description of excluded docs and wanted to confirm. 🙂  

Ignore if this is right!